package.json

```diff
@@ -122,8 +115,7 @@
     "stylelint-scss": "^3.21.0",
     "ts-patch": "^3.1.2",
     "typescript": "^5.4.5",
-    "webpack-bundle-analyzer": "^4.5.0",
-    "whatwg-fetch": "^3.6.2"
+    "webpack-bundle-analyzer": "^4.5.0"
```

Comment:
I think we can also remove the analyser stuff. If we configure this we should do it somewhere central to include in all apps.

Reply:
Yeah, forgot about this one